src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/GermlinePloidyAnnotatedTargetCollection.java

```diff
@@ -118,54 +132,45 @@ public int getTargetGermlinePloidyByGenotype(@Nonnull final Target target, @Nonn
      * @param targetList list of targets
      * @param contigAnnotsList list of contig ploidy annotations
      */
-    private void performValidityChecks(final List<Target> targetList, final List<ContigGermlinePloidyAnnotation> contigAnnotsList) {
+    private void performValidityChecks(@Nonnull final List<Target> targetList,
+                                       @Nonnull final List<ContigGermlinePloidyAnnotation> contigAnnotsList) {
         /* assert the lists are non-empty */
-        if (targetList.isEmpty()) {
-            throw new UserException.BadInput("Target list can not be empty");
-        }
-        if (contigAnnotsList.isEmpty()) {
-            throw new UserException.BadInput("Contig germline ploidy annotation list can not be empty");
-        }
+        Utils.validateArg(!Utils.nonNull(targetList, "Target list must be non-null").isEmpty(),
+                "Target list can not be empty");
+        Utils.validateArg(!Utils.nonNull(contigAnnotsList, "Contig ploidy annotation list must be non-null").isEmpty(),
+                "Contig germline ploidy annotation list can not be empty");
 
         /* assert targets have unique names */
         final Map<String, Long> targetNameCounts = targetList.stream()
                 .map(Target::getName)
                 .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
-        if (targetNameCounts.keySet().size() < targetList.size()) {
-            throw new UserException.BadInput("Targets must have unique names. Non-unique target names: " +
-                    targetNameCounts.keySet().stream()
-                            .filter(name -> targetNameCounts.get(name) > 1)
-                            .collect(Collectors.joining(", ")));
-        }
+        Utils.validateArg(targetNameCounts.keySet().size() == targetList.size(),
+                "Targets must have unique names. Non-unique target names: " + targetNameCounts.keySet().stream()
+                        .filter(name -> targetNameCounts.get(name) > 1)
+                        .collect(Collectors.joining(", ")));
 
         /* assert contigs are not annotated multiple times */
         final Map<String, Long> contigAnnotsCounts = contigAnnotsList.stream()
                 .map(ContigGermlinePloidyAnnotation::getContigName)
                 .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
-        if (contigAnnotsCounts.keySet().size() < contigAnnotsList.size()) {
-            throw new UserException.BadInput("Some contigs are multiply annotated: " +
-                    contigAnnotsCounts.keySet().stream()
+        Utils.validateArg(contigAnnotsCounts.keySet().size() == contigAnnotsList.size(),
+                "Some contigs are multiply annotated: " + contigAnnotsCounts.keySet().stream()
```

Comment:
These comments help me navigate through my own code quickly. I often just read the comments rather than the code :)